app/bundles/EmailBundle/Views/Email/list.html.php

```diff
@@ -168,7 +168,7 @@
                               title="<?php echo $view['translator']->trans('mautic.email.stat.leadcount.tooltip'); ?>">
                             <a href="<?php echo $view['router']->path(
                                 'mautic_contact_index',
-                                ['search' => $view['translator']->trans('mautic.lead.lead.searchcommand.emailpending').':'.$item->getId()]
+                                ['search' => $view['translator']->trans('mautic.lead.lead.searchcommand.email_pending').':'.$item->getId()]
```

Comment:
Do we really want to pass a translated strings? I needs to be encoded at least, but it is not good solution I think.
What if there will be na `&` character in translation?

Reply:
All of our "search commands" are translatable.

Reply:
And how is this working? What about special characters in translation? Like `&` mentioned above?

Reply:
Works because of the case statement that compares with same translation string (but also forces the English version).

For these use cases, I doubt & will be part of the translation. But it wouldn't hurt to url encode.